watson-developer-cloud/discovery-components
# fix: standardize eslintrc

#### What do these changes do/fix?

Cleanup up `.eslintrc` and standardize with main repo. Fix linter warnings and errors.

#### How do you test/verify these changes?

Run lint command. If you want to get crazy, replace the use of `lodash.get` with optional chaining -- notice that linter doesn't complain about that any more.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
